packages/js/product-editor/src/blocks/tab/edit.tsx

```diff
@@ -49,6 +69,31 @@ export function Edit( {
 				role="tabpanel"
 				className={ classes }
 			>
+				{ isOptionsNoticeVisible && (
+					<Notice
```

Comment:
This Notice is rather specific to Variations and I don't think it should be part of the tabs logic. I would propose either one of two things:
- Add support to tabs for any type of notice to display, we can potentially make use of the `notices` store for this, or have our own custom product editor store notice. Although this may have to be P2'd, as I am not sure we should allow the option for multiple notices at the top of each tab. As this could get overused ( cc @jarekmorawski  )
- Create a custom block just for the notice and add this at the top of each tab. This seems like the best and easiest approach for now.

Reply:
Ideally, we (as Woo) could show any number of notices in any tab, but it shouldn't be available to 3PDs.